packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -258,7 +258,7 @@ export interface Step {
 export interface WorkflowDefinition {
     workflowId: string;
     name: string;
-    description?: string;
+    description: string;
```

Comment:
By right, this is a breaking change. I also saw that this type is used in multiple other places:
![image](https://github.com/user-attachments/assets/01491996-6933-49aa-a0ec-a964d1bb9e18)

Did you verify all of them? Same goes for the TaskDefinition type below

Reply:
Yes, should be fine